### DISCUSSION TITLE
[@types/yup] property from missing in TestContext

### DISCUSSION THREAD

**Discussion by SalahAdDin** [DefinitelyTyped/DefinitelyTyped#59811] [Issues with a @types package]
I am working in a nested schema validation and i found [this issue](https://github.com/jquense/yup/issues/735) pretty useful and it leads us to [this other issue](https://github.com/jquense/yup/issues/398).

Inside of, i found the this [issue](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/49512), who is pretty related to my current bug:
```
Unsafe assignment of an `any` value.
Unsafe member access [1] on an `any` value.
Property 'from' does not exist on type 'TestContext<AnyObject>'
```
The problem is **form** clearly does exist, cause I'm getting it, but the property does not exist in the type.

In the aforementioned issue there is a workaround for this bug, but I didn't work for me.

It is noteworthy that that issue has two years, why does this issue still persist?

Thank you

[a comment by an automated account is not shown]

**Comment by mauricedb** [comment]
Hi @SalahAdDin,

Yup has been rewritten in TypeScript some time ago and these type definitions are no longer required as Yup now ships with it's own up to date type definitions. If the issue still persist with those I suggest creating an issue in the Yup repository.

Maurice

**Reply by BendingBender** [reply]
Will be removed in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61700.
